generator/template/.env.example

```diff
@@ -7,7 +7,9 @@ VERIFIER_API_URL="https://affinity-verifier.prod.affinity-project.org/api"
 CLOUD_WALLET_API_URL="https://cloud-wallet-api.prod.affinity-project.org/api"
 
 ISSUER_LOGIN="issuer@affinidi.com"
-ISSUER_PASSWORD="test"
+# NOTE: this is only an usage example
+# hash should be stored in DB
+ISSUER_HASH_PASSWORD="\$2b\$10\$Po7rWM3n2WirHumG/sqmUuCZQauXoWAekclDKiYYrnvuWBAaTTt9i"
```

Comment:
current `ISSUER_HASH_PASSWORD` encrypted with salt but no need for salt as an additional parameter because `bcrypt` can compare text password and hash without salt even if it already used